src/styles/calendarDay.scss

```
@@ -0,0 +1,42 @@
+.calendar__day {
+  height: $squareSide;
+  width: $squareSide;
+
+  text-align: center;
+
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 330%;
+
+  background-color: #eee;
```

don't forget to create variables for all colors

```suggestion
  background-color: $c-black;
```

---

I've used this color only once, do I really need a variable?

---

You should get used to create variables for each color, bc in the production you will do this)